xml/System.IO/Path.xml

```diff
@@ -1402,16 +1402,48 @@ The following example defines a variable, `basePath`, to represent an applicatio
         <Parameter Name="path" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="path">The path from which to obtain root directory information.</param>
+        <param name="path">A read-only span of characters containing the path from which to obtain root directory information.</param>
         <summary>Gets the root directory information from the path contained in the specified character span.</summary>
-        <returns>A character span containing the root directory of `path`.</returns>
+        <returns>A read-only span of characters containing the root directory of <paramref name="path" />.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
 
+This method does not verify that the path or file name exists.
+
 Unlike the string overload, this method doesn't normalize directory separators.
 
+A `ReadOnlySpan<System.Char>` is "effectively empty" if:
+
+- In Windows, calling `IsEmpty` on this span of characters returns `true`, or all its characters are spaces (' ').
+- In Unix, calling `IsEmpty` on this span of characters returns `true`.
+
+Possible patterns for the string returned by this method are as follows:
+
+- `null` (`path` was null or an empty string).
+
+- An empty string (`path` specified a relative path on the current drive or volume).
+
+- "\" (Unix: `path` specified an absolute path on the current drive).
+
+- "X:" (Windows: `path` specified a relative path on a drive, where X represents a drive or volume letter).
+
+- "X:\" (Windows: `path` specified an absolute path on a given drive).
+
+- "\\\ComputerName\SharedFolder" (Windows: a UNC path).
+
+- "\\\\\?\C:" (Windows: a DOS device path, supported in .NET Core 1.1 and later versions and in .NET Framework 4.6.2 and later versions)
+ 
+For more information on file paths on Windows, see [File path formats on Windows systems](~/docs/standard/io/file-path-formats.md). For a list of common I/O tasks, see [Common I/O Tasks](~/docs/standard/io/common-i-o-tasks.md).
+
+## Examples
```

Comment:
The examples section should be deleted, since it illustrates the wrong overload.

Reply:
```suggestion
```